hs_core/views/resource_folder_hierarchy.py

```diff
@@ -27,21 +27,21 @@ def data_store_structure(request):
     """
     res_id = request.POST.get('res_id', None)
     if res_id is None:
-        return HttpResponse(content_type='application/json', status=400)
+        return HttpResponse('Bad request - resource id is not included', status=400)

```

Comment:
Since the front end expects a default ```text/html``` content-type for an error response, and the status code is set on HttpResponse object, the default Django ```text/html``` content type for HttpResponse object can be directly used without explicitly setting it, which is what has been done across our code base.